### PR TITLE
LPS-171181 Fix CI routine

### PR DESCRIPTION
**Pull request by luanakarla**

### CODE REVIEW

**Comment by luanakarla** [comment]
ci:test:sf

**Comment by luanakarla** [comment]
ci:test:sf

**Comment by luanakarla** [comment]
ci:test:osb-site-initializer-customer-portal

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2AnalyticsCloudSetup.macro [hunk 1]
```diff
@@ -44,13 +44,13 @@ definition {
...
-			position = "${position}",
-			status = "${status}");
-
 		CP2DXPCloudSetup.deleteEntryCloudEnvironments(
 			entry = "${entry}",
 			portlet = "Analytics Cloud Workspaces");
+
+		CustomerPortal.clearActivationStatus(
+			position = "${position}",
+			status = "${status}");
```
Comment (barrosdennys) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2AnalyticsCloudSetup.macro:53]: the macro says clear and delete but the order here is deleting then it clears. Is it correct?

Comment (luanakarla) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2AnalyticsCloudSetup.macro:53]: oh yes, I hadn't realized. I'll change.

**Comment by luanakarla** [comment]
ci:test:osb-site-initializer-customer-portal

**Comment by barrosdennys** [comment]
ci:forward

**Pull request closed without merging**